plaso/analysis/browser_search.py

```diff
@@ -175,6 +177,28 @@ def _ExtractGoogleSearchQuery(self, url):
 
     return line.replace('+', ' ')
 
+  def _ExtractYahooSearchQuery(self, url):
+    """Extracts a search query from a Yahoo search URL.
```

Comment:
Technically this function extract a search query from any URL that contains `q=`. Is this the intended functionality? If so maybe reflect this in the docstring, and maybe this method can be merged with other methods that do the same?

Reply:
I'm OK with pushing now, if this is urgently needed and create and issue and follow up CL to generalize this.

Reply:
I've created: https://github.com/log2timeline/plaso/issues/2216